src/components/Menu/Menu.module.css

```diff
@@ -21,4 +21,30 @@
     display: flex;
     align-items: center;
   }
+
+  ul[role*=menu] li div[role*=presentation] {
+    padding-left: calc(var(--spacing-margin-xs, 4) * 1px + var(--shape-size-lg, 24) * 1px) !important;
+  }
+
+  ul[role*=menu] li ul[role*=menu] li div[role*=presentation] {
+    padding-left: calc(var(--spacing-margin-xs, 4) * 1px + var(--shape-size-lg, 24) * 1px + var(--spacing-padding-md, 12) * 1px) !important;
+  }
+
+  ul[role*=menu] li ul[role*=menu] li ul[role*=menu] li div[role*=presentation] {
+    padding-left: calc(var(--spacing-margin-xs, 4) * 1px + var(--shape-size-lg, 24) * 1px + var(--spacing-padding-md, 12) * 2px) !important;
+  }
```

Comment:
ma così funziona solo per i primi X livelli, che ok sicuramente ci andrà bene, però non capisco perchè non eredita il padding di quello di sopra a questo punto. Così fatto il problema si ripropone

Reply:
I have not found an infinitely extensible solution. However, the implementation works for the first 4 levels of indentation, which is more than enough to cover all our present and future cases:

![Screenshot from 2024-02-28 13-59-53](https://github.com/mia-platform/design-system/assets/125553486/65c7cb5f-f936-4c8f-a6a0-9d89ac8f8eae)

Reply:
Solved in https://github.com/mia-platform/design-system/pull/355 ✅